src/libprojectM/ProjectM.hpp

```diff
@@ -343,7 +343,7 @@ class ProjectM
     };
 
 
-    inline Pcm* pcm()
+    inline Pcm& pcm()
```

Comment:
`inline` should be removed from all member functions in the codebase.

Reply:
You're right. Doesn't make a lot of sense nowadays, as compilers tend to just ignore it anyway.